cmd/ocm-backplane/login/login.go

```diff
@@ -81,8 +100,36 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	logger.Debugf("Checking Backplane Version")
 	utils.CheckBackplaneVersion(cmd)
 
-	logger.Debugf("Extracting Backplane Cluster ID")
-	// Get The cluster ID
+	logger.Debugf("Extracting Backplane configuration")
+	// Get Backplane configuration
+	bpConfig, err := config.GetBackplaneConfiguration()
+	if err != nil {
+		return err
+	}
+	logger.Debugf("Backplane Config File Contains is: %v \n", bpConfig)
+  
+  logger.Debugf("Extracting Backplane Cluster ID")
+	// Currently go-pagerduty pkg does not include incident id validation.
+	if args.pd != "" {
+		pdClient, err := pagerduty.NewWithToken(bpConfig.PagerDutyAPIKey)
+		if err != nil {
+			return fmt.Errorf("could not initialize the client: %w", err)
+		}
+		if strings.Contains(args.pd, "redhat.pagerduty.com/incidents/") {
```

Comment:
I don't think it's good idea to expose `redhat.pagerduty.com` internal PD URL into public repo 
may be 
```
if strings.Contains(args.pd, "/incidents/")
```